awslabs/eventbridge-kafka-connector
# feat: support global endpoints

Description
-----------

This feature adds support for EventBridge Global Endpoints. To use EventBridge Global Endpoints, `PutEvents` has an [`EndpointId`](https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_PutEvents.html#API_PutEvents_RequestSyntax) parameter. 

When setting `aws.eventbridge.eventbus.global.endpoint.id` (bus setting), which must not be confused with `aws.eventbridge.endpoint.uri` (SDK setting), the global endpoint specified using `abcde.xyz` syntax is used.

Test Steps
-----------

Tested existing connector using an event bus without Global endpoints works.
Tested new feature against a Global Endpoint specified with `aws.eventbridge.endpoint.uri` in the connector configuration.

Checklist:
----------




Related Issue
-----------

Closes: #67


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
